components/script/dom/htmlimageelement.rs

```diff
@@ -1132,26 +1185,26 @@ impl HTMLImageElement {
                 let relevant_mutation = this.generation.get() != generation;
                 // Step 15.1
                 if relevant_mutation {
-                    this.abort_request(State::Unavailable, ImageRequestPhase::Pending);
+                    let mut pending_opt = this.pending_request.borrow_mut();
+                    *pending_opt = None;
```

Comment:
We could write this (and other similar uses) as `*this.pending_request.borrow_mut() = None;`. Up to you if you think that's an improvement.

Reply:
That's definitely better.  Will change.